docker/bin/zookeeperStart.sh

```diff
@@ -71,23 +81,9 @@ set -e
 
 # Determine if there is an ensemble available to join by checking the service domain
 set +e
-getent hosts $DOMAIN  # This only performs a dns lookup
+nc -v -w2 $CLIENT_HOST $CLIENT_PORT  # This only performs a dns lookup
```

Comment:
why?

Reply:
I belive nc approach is solving the same problem, a bit more elegant, by getting read of the while loop. In the end we want to be able to connect to the SEED node. Am I missing something?

Reply:
This PR is not yet final, was made to get zk expansion work.